### PR TITLE
[CI] Define triggers properly.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by stephen-hawley — APPROVED** [review]

File: devops/automation/azure-pipelines.yml [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-
 trigger:
-- '*'
+  branches:
+    include:
+    - '*'
 
 pr:
-- '*'
+  autoCancel: true
```
Comment (tj-devel709) [inline devops/automation/azure-pipelines.yml:10]: Should we also have this in macios or *do we have this in the macios one?

**Review by tj-devel709 — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Failures are unrelated.

File: devops/automation/azure-pipelines.yml (hunk 1 above)
Comment (mandel-macaque) [reply devops/automation/azure-pipelines.yml:10]: What do you mean?

Comment (tj-devel709) [reply devops/automation/azure-pipelines.yml:10]: I'll message you!

**Pull request merged**